.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+## Chromatic
+https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
```

Comment:
Can we have a comment above this so that PR authors know to leave it alone? Something like:

```md
<!-- This `{{head.ref}}` is a placeholder for a CI job - it will be updated automatically -->
```